docs/getting-started/create-flet-app.md

```diff
@@ -42,11 +42,13 @@ The command will create the following directory structure:
 ```
 
 :::note
-Original `pyproject.toml` created by `uv init` or `poetry init` will be replaced with the one from Flet app template.
+1. Original `pyproject.toml` created by `uv init` or `poetry init` will be replaced with the one from Flet app template.
+2. In case you encounter the error `Error creating the project from a template: 'git' is not installed.`, it means that you don't have _Git_ installed. Please visit [git-scm.com/downloads](https://git-scm.com/downloads) and install the latest version of _Git_. To verify your installation, type ‘git’ on Command Prompt.
```

Comment:
```suggestion
2. In case you encounter the error `Error creating the project from a template: 'git' is not installed.`, it means that you don't have _Git_ installed. Please visit [git-scm.com/downloads](https://git-scm.com/downloads) and install the latest version of _Git_. To verify your installation, type `git` in the terminal.
```